integration/test_redis_specific.py

```diff
@@ -68,3 +86,6 @@ def test_redis_driver_join(redis):
         {'col1': 1, 'col2': '2', 'name': 1234}]
 
     assert termination_result == expected_result
+
+    for key in driver.redis.scan_iter(f'*storey:{table_name}*'):
+        driver.redis.delete(key)
```

Comment:
If the point is cleanup, better put this in teardown or in a finally clause.

Reply:
ok